lib/assessment.js

```diff
@@ -4,32 +4,57 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
+// Write a function that takes zero or more arguments
+// and returns the minimum from that set
+// assume that all the argument are numbers
+    // var min = function min(nums) {
+    //     var length = arguments.length;
+    //     for(i=0; i<length;i++){
+    //         I don't know the syntax to break this up!
+    //         { '0': 10, '1': 12, '2': 14 }
+    //         How do I pull out 10 with a key of 0?
+    //     }
+    //     return arguments;
+    // };
 
-};
+    // min(10,12,14)
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
 var sum = function sum() {
-
+    var total=0;
+    var i = 0;
+    while(i < arguments.length) {
+        total =+ arguments[i] + i + 1;
+        i++;
+    }
```

Comment:
I would probably have suggested using a `for` loop here, but using a `while` is ok. 

However, it should be `+=`, not `=+`. Additionally, it's not clear why you're adding `+ i + 1` each time you go through the `while` loop.